pinot-common/src/main/java/org/apache/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -73,6 +73,8 @@ public static class CustomHeaders {
     public static final String UPLOAD_TYPE = "UPLOAD_TYPE";
     public static final String REFRESH_ONLY = "REFRESH_ONLY";
     public static final String DOWNLOAD_URI = "DOWNLOAD_URI";
+
+    public static final String MOVE_SEGMENT_TO_DEEP_STORE = "MOVE_SEGMENT_TO_DEEP_STORE";
```

Comment:
Consider changing it to `COPY_SEGMENT_TO_DEEP_STORE` and add some comments stating that is is for metadata push only